test/test_collector.py

```diff
@@ -1740,6 +1758,161 @@ def test_param_sync(self, give_weights, collector, policy_device, env_device):
             col.shutdown()
 
 
+class TestBringReset:
```

Comment:
@matteobettini here is the place where we test the functionality of brining resets at the root.
In there, there's a check that if we have a partially populated list of resets we raise an exception